config/routes.rb

```diff
@@ -42,6 +42,10 @@
   resources :packages do
     put :activate, on: :member
     put :deactivate, on: :member
+    if Rails.env.development? || Rails.env.test?
```

Comment:
See if you can remove these routing conditions and handle it all with can-can + the pl/sql settings user permissions?